velero/spectrum-fusion/recipes/no-olm/core/child-zen-recipe-2.10.0.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: spp-data-protection.isf.ibm.com/v1alpha1
+kind: Recipe
+metadata:
+  labels:
+    dp.isf.ibm.com/parent-recipe: cpfs-parent-recipe
+    dp.isf.ibm.com/parent-recipe-namespace: <parent recipe namespace>
+  name: zen-child-<child recipe namespace>
+  namespace: <child recipe namespace>
+spec:
+  appType: common-service
+  groups:
+    - labelSelector: foundationservices.cloudpak.ibm.com=zen5-data
+      name: zen-volume
+      includedNamespaces:
+        - <child recipe namespace>
+      type: volume
+    - includeClusterResources: true
+      includedResourceTypes:
+        - zenservices.zen.cpd.ibm.com
+        - customresourcedefinitions.apiextensions.k8s.io
+      labelSelector: foundationservices.cloudpak.ibm.com=zen
```

Comment:
The label-common-service.sh script labels the zenservice as zen-chart, we should probably either update the label-common-service.sh script to change what it labels in no olm mode or update this value to match what it is already doing

Reply:
the zenservice crd is also labeled as zen-cluster so it is also not covered here. If neither are backed up we get a full failure